### PR TITLE
[ROX-17732] Remove unused CVE datastore

### PR DESCRIPTION
**Pull request by md2119**
## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### CODE REVIEW

File: central/cve/node/datastore/datastore_impl_test.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//go:build sql_integration
```
Comment (connorgorman) [inline central/cve/node/datastore/datastore_impl_test.go:1]: Why does this need this tag?

File: central/namespace/datastore/datastore.go [hunk 2]
```diff
@@ -54,11 +46,13 @@ type DataStore interface {
...
-		deployments:     deploymentDataStore,
-		namespaceRanker: namespaceRanker,
+		store:             nsStore,
+		indexer:           indexer,
+		deployments:       deploymentDataStore,
+		namespaceRanker:   namespaceRanker,
+		formattedSearcher: formatSearcherV2(indexer, namespaceRanker),
 	}
+
 	ds.formattedSearcher = formatSearcherV2(indexer, namespaceRanker)
```
Comment (connorgorman) [inline central/namespace/datastore/datastore.go:56]: remove?

**Review by connorgorman — CHANGES_REQUESTED** [review]
a few comments, but the main one is how can we just delete the service endpoint?

File: central/cve/node/datastore/datastore_impl_test.go (hunk 1 above)
Comment (md2119) [reply central/cve/node/datastore/datastore_impl_test.go:1]: It is a postgres ds test

File: central/namespace/datastore/datastore.go (hunk 2 above)
Comment (md2119) [reply central/namespace/datastore/datastore.go:56]: No, there are some wrapper still in use in postgres. But again, focusing on cve datastore in this PR.

File: central/cve/node/datastore/datastore_impl_test.go (hunk 1 above)
Comment (connorgorman) [reply central/cve/node/datastore/datastore_impl_test.go:1]: it doesn't use a real postgres instance so doesn't need it

File: central/namespace/datastore/datastore.go (hunk 2 above)
Comment (dashrews78) [reply central/namespace/datastore/datastore.go:56]: isn't `formattedSearcher: formatSearcherV2(indexer, namespaceRanker)` and `ds.formattedSearcher = formatSearcherV2(indexer, namespaceRanker)` duplication?

**Comment by md2119** [comment]
> a few comments, but the main one is how can we just delete the service endpoint?

The service has been deprecated and cannot be use with postgres.

File: central/cve/node/datastore/datastore_impl_test.go (hunk 1 above)
Comment (md2119) [reply central/cve/node/datastore/datastore_impl_test.go:1]: Ah, it is using mock stores

File: central/namespace/datastore/datastore.go (hunk 2 above)
Comment (md2119) [reply central/namespace/datastore/datastore.go:56]: I see what you mean

**Review by connorgorman — APPROVED** [review]

**Review by dashrews78 — APPROVED** [review]

**Pull request merged**